dts/bindings/dma/st,stm32-dma.yaml

```diff
@@ -0,0 +1,51 @@
+#
+# Copyright (c) 2019, Song Qiang <songqiang1304521@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: STM32 DMA
+version: 0.1
+
+description: >
+    This is a representation of the STM32 DMA nodes
+
+inherits:
+  !include dma.yaml
+
+properties:
+    compatible:
+      constraint: "st,stm32-dma"
+
+    reg:
+      type: array
+      description: register base address
+      generation: define
+      category: required
+
+    interrupts:
+      type: array
+      category: required
+      description: required irq number and irq priorities
+      generation: define
+
+    support-m2m:
```

Comment:
this looks like it should just be a boolean, ie if the property exists than its true, if not its false.

Reply:
Right